sign/sign.go

```diff
@@ -299,6 +394,40 @@ func (s *Signer) IsImageSigned(imageRef string) (bool, error) {
 	return len(signatures) > 0, nil
 }
 
+// IsFileSigned takes an path reference and retrusn true if there is a signature
+// available for it. It makes no signature verification, only checks to see if
+// there is a TLog to be found on Rekor.
+func (s *Signer) IsFileSigned(ctx context.Context, ko cliOpts.KeyOpts, path string) (bool, error) { // nolint: gocritic
```

Comment:
Let's either keep it private or abstract (hide) the externally facing API `ko cliOpts.KeyOpts`.

Reply:
Would it be an option to add ko to Singer struct as a internal var?

Reply:
Right now we create the instance there: https://github.com/kubernetes-sigs/release-sdk/pull/68/files#diff-9c6c93e3b9d57b2aacb32e9b81617b1d8e41614365745f2e1845f23120605b23R310-R313

Can we move it into this function to avoid having the argument at all?

Reply:
Yes we can. We can create an instance just like we did in `VerifyFile` and remove the argument `ko` from `IsFileSigned`. All input is available through signer options or `cliOpts.Defaults`.